### DISCUSSION TITLE
ssl setup db2 driver

### DISCUSSION THREAD

**Discussion by bardissi** [dbeaver/dbeaver#12304] [Q&A]
hey mates, 
newbie question: setting up Connection to db2 LUW.
getting the following error upon testing connection
Connect Timed out. ERRORCODE=-4499 SQLSTATE=08001

i am guessing i need to point to ssl file : xxx.jks
i have the file, but i dont have ssl tab to point to

using macbook
please advise

**Comment by mLokesh834** [comment]
Hi 
https://www.ibm.com/docs/en/fci/1.1.0?topic=application-connecting-db2-over-ssl-port
1. In Dbeaver go to  Driver Properties.
2. Left side you see the + symbol
3.Add the Properties .
Under Properties, select the Optional tab and add the following property values:
sslTrustStoreLocation: Specify the path and file name to the*************.jks keystore file on your local system.
sslTrustStorepassword: Specify the password for the SSL keystore 
sslConnection: Specify true.

Thanks
